### PR TITLE
[stable/openebs] update charts to 1.7.0 release

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@openebs.io>

 No

#### What this PR does / why we need it:
PR updates the openebs version with latest 1.7.0 release
- Updated README
- Updated chart version
- Updated values.yaml

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/openebs/values.yaml [hunk 1]
```diff
@@ -38,10 +38,15 @@ apiserver:
 defaultStorageConfig:
   enabled: "true"
 
+persistentStoragePath:
```
Comment (kmova) [inline stable/openebs/values.yaml]: Let us call this - `varDirectoryPath`. The word persistent will get confusing with the actual storage location.  

The comment can say: "This directory is used by the OpenEBS to store debug information and so forth that are generated in the course of running OpenEBS containers."

Comment (prateekpandey14) [reply stable/openebs/values.yaml]: Done

**Review by akhilerm — APPROVED** [review]
/lgtm

**Review by kmova — APPROVED** [review]
/lgtm
/approved

**Pull request merged**